tests/reference_scores/reference_task_scores.py

```diff
@@ -615,5 +615,59 @@
         "lighteval|bigbench:tracking_shuffled_objects_five_objects|3|0": {"acc": 0.2000, "acc_stderr": 0.1333},
         "lighteval|bigbench:tracking_shuffled_objects_seven_objects|3|0": {"acc": 0.3000, "acc_stderr": 0.1528},
         "lighteval|bigbench:tracking_shuffled_objects_three_objects|3|0": {"acc": 0.4000, "acc_stderr": 0.1633},
+        "lighteval|agieval:_average|0|0": {
```

Comment:
those come from YALL or you computed them with lighteval ?

Reply:
These our own results on this model, which is not in YALL iirc - but I tested on a range of models from YALL and we are always within stderr range.